.github/workflows/ci.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           - displayTargetName: windows-2019
             os: windows
             runs_on: windows-2019
+            pip_option: --break-system-packages
```

Comment:
can we use the `--user` parameter to install to a local `%APPDATA%\Python` path instead of modifying the system python package? It's for CI so it probably doesn't matter...

Reply:
I think it can.  For this CI, I wanted to reduce OS depends code so, used the nasty option(--break-system-packages) for that.